### PR TITLE
Added vendorlib and version number requirement

### PR DESCRIPTION
**Pull request by tknippenberg**

### CODE REVIEW

File: content/en/docs/appstore/sharing-content/_index.md [hunk 1]
```diff
@@ -76,7 +76,9 @@ To develop modules and submit them to the Marketplace, follow these guidelines:
 
 * Create a folder named **USE_ME** and add the microflows and pages that are relevant for the user
 * Create an empty folder with the version number as its name, which will appear in Studio Pro's App Explorer
-* If you add any **userlib** .*jar* files, make sure they are accompanied by a blank **MyModule.RequiredLib** file so that users know where the .*jar* files come from
+* Java dependencies
+    * In versions Mendix 10.6.0 and above, use [managed dependencies]{/refguide/managed-dependencies/} where possible. For versions below 10.6, ensure that Java dependencies are put in the `userlib` folder. You should also put any [unmanaged dependencies](/refguide/managed-dependencies/#unmanaged) (that is, non-publicly-available `.jar` files) in the `userlib` folder.
```
Comment (MarkvanMents) [inline content/en/docs/appstore/sharing-content/_index.md]: This version seems to conflict with what we have in our [Managed Dependencies](https://docs.mendix.com/refguide/managed-dependencies/#1-introduction) documentation
image
I think they should be aligned?

File: content/en/docs/appstore/sharing-content/_index.md [hunk 2]
```diff
@@ -76,7 +76,9 @@ To develop modules and submit them to the Marketplace, follow these guidelines:
...
+        * When putting `.jar` files in the `userlib` folder, make sure the name includes a version number (for example, `org.apache.commons.io-2.3.0.jar`) and is accompanied by a blank `{jarfile}.MyModule.RequiredLib` file so that users know where the .*jar* files come from (for example, `org.apache.commons.io-2.3.0.jar.MyModule.RequiredLib`)
```
Comment (MarkvanMents) [inline content/en/docs/appstore/sharing-content/_index.md]: Should **MyModule** be a placeholder or is it always **MyModule**?
The doc doesn't use **MyModule** anywhere else, so it isn't clear to me.
Would it better to be `{jarfile}.{module name}.RequiredLib`?

**Review by MarkvanMents — CHANGES_REQUESTED** [review]
@tknippenberg 
Hi Tom,
Thanks for the PR.
I have a couple of questions which I think could make the documentation clearer. Can you let me know what you think?
Thanks.
Mark

File: content/en/docs/appstore/sharing-content/_index.md (hunk 1 above)
Comment (tknippenberg) [reply content/en/docs/appstore/sharing-content/_index.md]: Hm let's call it early Monday morning, shall we? ;-) 10.3.0 is correct!

File: content/en/docs/appstore/sharing-content/_index.md (hunk 2 above)
Comment (tknippenberg) [reply content/en/docs/appstore/sharing-content/_index.md]: Placeholders is indeed better!

Either {jarfile}.{module_name}.RequiredLib or {jarfile-including-version}.{module_name}.RequiredLib could work. Thanks!

File: content/en/docs/appstore/sharing-content/_index.md (hunk 1 above)
Comment (MarkvanMents) [reply content/en/docs/appstore/sharing-content/_index.md]: Done: https://github.com/mendix/docs/pull/7461/commits/1383898c0f578998cbbd535ec7e664ee8d8db39d

File: content/en/docs/appstore/sharing-content/_index.md (hunk 2 above)
Comment (MarkvanMents) [reply content/en/docs/appstore/sharing-content/_index.md]: Done https://github.com/mendix/docs/pull/7461/commits/1383898c0f578998cbbd535ec7e664ee8d8db39d

**Review by MarkvanMents — APPROVED** [review]
Hi Tom,
Thanks for the quick response.
Approving. I guess this can be merged immediately.
Mark
@tknippenberg

**Pull request merged**